src/main/java/dudu/task/TaskList.java

```diff
@@ -0,0 +1,107 @@
+package dudu.task;
+
+import java.util.ArrayList;
+
+import dudu.exception.DuduException;
+import dudu.exception.TaskNumRangeException;
+
+/**
+ * Task list class for storing tasks
+ */
+public class TaskList {
+    private final ArrayList<Task> list;
```

Comment:
perhaps a variable name stating the contents of the list would be helpful to others who are not familiar with your code? example "taskList".